tests/acceptance/test_sasl.py

```diff
@@ -0,0 +1,69 @@
+# MIT License
+# Copyright (c) 2020-2024 Pau Freixes
+
+import asyncio
+import logging
+import os
+import sys
+from pathlib import Path
+
+import pytest
+
+from emcache import MemcachedHostAddress, create_client
+from emcache.client_errors import AuthenticationError
+
+pytestmark = pytest.mark.asyncio
+
+logging.basicConfig(level=logging.DEBUG)
+
+with open(Path(os.path.dirname(__file__)) / "data" / "auth_pwd.txt", "r") as f:
+    username, password = f.readline().rstrip().split(":")
+
+
+@pytest.fixture()
+async def authed_client(event_loop):
+    client = await create_client(
+        [MemcachedHostAddress("localhost", 11214)], timeout=1.0, username=username, password=password
+    )
+    try:
+        yield client
+    finally:
+        await client.close()
+
+
+class TestAuth:
+    @pytest.mark.skipif(sys.platform == "darwin", reason="This server is not built with SASL support.")
```

Comment:
Unless Im missing something we can run all of these tests for Mac too, we need to add the new server [here](https://github.com/emcache/emcache/blob/master/.github/workflows/ci.yml#L51)

Also lets add it [here](https://github.com/emcache/emcache/blob/master/.github/workflows/release.yml#L44)

Reply:
sasl does not work on mac. of course, you can try again
https://github.com/memcached/memcached/issues/1143

Reply:
macos now not support sasl(
https://github.com/ArtemIsmagilov/emcache/actions/runs/9767878503/job/26964160619

Reply:
Oks, can we make the comment a bit more clear? like "SASL not supported by memcache running in Mac"